src/quacc/schemas/ase.py

```diff
@@ -437,3 +443,55 @@ def _summarize_ideal_gas_thermo(
     )
 
     return atoms_metadata | inputs | results
+
+
+def _summarize_harmonic_thermo(
+    ht: HarmonicThermo, temperature: float = 298.15, pressure: float = 1.0
+) -> ThermoSchema:
+    """
+    Get tabulated results from an ASE HarmonicThermo object and store them in a
+    database-friendly format.
+
+    Parameters
+    ----------
+    ht
+        ASE HarmonicThermo object.
+    temperature
+        Temperature in Kelvins.
+    pressure
+        Pressure in bar.
+
+    Returns
+    -------
+    ThermoSchema
+        Dictionary representation of the task document
+    """
+    settings = get_settings()
+
+    inputs = {
+        "parameters_thermo": {
+            "temperature": temperature,
+            "pressure": pressure,
+            "vib_freqs": [e / units.invcm for e in ht.vib_energies],
+            "vib_energies": ht.vib_energies.tolist(),
+            "n_imag": ht.n_imag,
+        }
+    }
+
+    results = {
+        "results": {
+            "energy": ht.potentialenergy,
+            "helmholtz_energy": ht.get_helmholtz_energy(
+                temperature, verbose=settings.DEBUG
+            ),
+            "internal_energy": ht.get_internal_energy(
+                temperature, verbose=settings.DEBUG
+            ),
+            "entropy": ht.get_entropy(temperature, verbose=settings.DEBUG),
+            "zpe": ht.get_ZPE_correction(),
+        }
+    }
+
+    atoms_metadata = {}
+
+    return atoms_metadata | inputs | results
```

Comment:
If we stick with `{}` then there's no need to concatenate it to the other dictionaries. However, I think we should probably be generating an `atoms_metadata` like what was done for the ideal gas case because otherwise the output dictionary from a run will not have the resulting `Atoms` object in it, which means the user can't do a subsequent job. All outputs in quacc need to support doing `results["atoms"]` basically.

I'm guessing this was done because `HarmonicThermo` does not have an `.atoms` attribute. I would suggest adding an `atoms: Atoms = None` keyword argument to `summarize_vib_and_thermo` that you can use to pass in the output `Atoms` object in your VASP recipe, where it will be turned into metadata here.

Make the `Atoms` object a required positional argument to `_summarize_harmonic_thermo` though so other developers don't forget to pass it in.